cogs5e/utils/checkutils.py

```diff
@@ -100,10 +100,18 @@ def run_save(save_key, caster, args, embed):
         embed.title = f"An unknown creature makes {a_or_an(save_name)}!"
     else:
         embed.title = f'{caster.get_title_name()} makes {a_or_an(save_name)}!'
-
-    # ieffect -sb
+    
+    # ieffect handling
     if isinstance(caster, init.Combatant):
+        # -sb
         args['b'] = args.get('b') + caster.active_effects('sb')
+        # -sadv/sdis
+        if not args.adv(boolwise=False, ephem=True):
```

Comment:
Checking for the presence of advantage with `ephem=True` here would consume a use of any ephemeral argument. Is the purpose of this line to have any given adv/dis argument completely override effects rather than being mixed in?

Reply:
(If so, I believe `'adv' in args or 'dis' in args` would do the trick without consuming ephemeral uses)

Reply:
Covered by force adding, given how argparse.adv() function only reads last.